tests/tools/Makefile

```diff
@@ -17,13 +20,9 @@ clean:
 
 .PHONY: $(BUILDDIR)
 $(BUILDDIR): \
-	$(BUILDDIR)/ginkgo \
 	$(BUILDDIR)/go-md2man \
 	$(BUILDDIR)/golangci-lint
 
-$(BUILDDIR)/ginkgo:
-	$(GO_BUILD) -o $@ ./vendor/github.com/onsi/ginkgo/ginkgo
```

Comment:
Should this be removed or changed to v2?

https://github.com/containers/buildah/pull/6537/files#diff-2835dca8e92a9a76c472158185ece8e624aa67c59b8e6b9b41574425684fe4bcR25